packages/webapp/.env

```diff
@@ -1,12 +1,12 @@
-NEXT_PUBLIC_API_URL=http://api.local.com
-#NEXT_PUBLIC_API_URL=https://api.daily.dev
-NEXT_PUBLIC_SUBS_URL=ws://api.local.com/graphql
-NEXT_PUBLIC_DOMAIN=local.com
+NEXT_PUBLIC_API_URL=https://api.local.fylla.dev
```

Comment:
probably need to add `fylla.dev` origin for hemidall and any service for CORS?

Reply:
Already added

Reply:
👌